src/modules/SdnDiag.Utilities/private/Remove-PSRemotingSession.ps1

```diff
@@ -8,37 +8,54 @@ function Remove-PSRemotingSession {
 
     [CmdletBinding()]
     param (
-        [Parameter(Mandatory = $true)]
+        [Parameter(Mandatory = $false)]
```

Comment:
in which condition we pass no ComputerName for a PSRemoteSession?

Reply:
@luyaof during the module import. This just ensures that during Import-Module, when we load the SdnDiagnostics.psm1, we do a cleanup within the runspace.

Seen a couple instances where PS window is left open and the session(s) previously never died out. So if you perform an update and re-import the module, it will use existing pssessions that are using a different version of sdndiagnostics.